lib/gears/color.lua

```diff
@@ -234,27 +254,9 @@ function color.create_pattern_uncached(col)
     return color.create_solid_pattern(col)
 end
 
---- Create a pattern from a given string.
--- This function can create solid, linear, radial and png patterns. In general,
--- patterns are specified as strings formatted as"type:arguments". "arguments"
--- is specific to the pattern used. For example, one can use
--- "radial:50,50,10:55,55,30:0,#ff0000:0.5,#00ff00:1,#0000ff"
--- Alternatively, patterns can be specified via tables. In this case, the
--- table's 'type' member specifies the type. For example:
--- { type = "radial", from = { 50, 50, 10 }, to = { 55, 55, 30 },
---   stops = { { 0, "#ff0000" }, { 0.5, "#00ff00" }, { 1, "#0000ff" } } }
--- Any argument that cannot be understood is passed to @{create_solid_pattern}.
---
--- Please note that you MUST NOT modify the returned pattern, for example by
--- calling :set_matrix() on it, because this function uses a cache and your
--- changes could thus have unintended side effects. Use @{create_pattern_uncached}
--- if you need to modify the returned pattern.
--- @see create_pattern_uncached, create_solid_pattern, create_png_pattern,
---   create_linear_pattern, create_radial_pattern
--- @param col The string describing the pattern.
--- @return a cairo pattern object
+--- Create a pattern from a given string., same as gears.color
```

Comment:
s/string.,/string/
Punctuation/dot at the end, please.  (move the dot ;)
You should also refer to `gears.color` (not only by `@see` - but also add backticks in the doc).